doc/api/assert.md

```diff
@@ -792,19 +1201,39 @@ Tests shallow, coercive equality between the `actual` and `expected` parameters
 using the [Abstract Equality Comparison][] ( `==` ). `NaN` is special handled
 and treated as being identical in case both sides are `NaN`.
 
-```js
-const assert = require('assert');
+```mjs
+import {
+  equal,
+} from 'assert';
```

Comment:
Why not use default import here? `assert.equal` is more readable than `equal` IMHO.
```suggestion
import assert from 'assert';
```

Reply:
Part style, part being opinionated. I've also been seeing a lot more user code favoring named imports lately. So really just a preference but nothing I'd lay down in the road for